core/src/scene/sceneLoader.cpp

```diff
@@ -64,6 +64,56 @@ void printFilters(const SceneLayer& layer, int indent){
     }
 };
 
+void SceneLoader::applyGlobalProperties(Node& node, Scene& scene) {
+    switch(node.Type()) {
+    case NodeType::Scalar:
+        {
+            std::string key = node.Scalar();
+            if (key.compare(0, 7, "global.") == 0) {
+                key.replace(0, 7, "");
+                key = std::regex_replace(key, std::regex("\\."), DELIMITER);
```

Comment:
Aha, can fix that, previously was replace a "." with "_$_", hence `regex_replace` was required.